File: spoty-android-app/spoty/Data/Models/Rating.cs
using Newtonsoft.Json;

namespace Spoty.Data.Models
{
    public class Rating
    {
        [JsonProperty("Grade")]
        public int Grade { get; }
        [JsonProperty("Feedback")]
        public string Feedback { get; }
        [JsonProperty("IdUserAccount")]
        public int IdUserAccount { get; }
        [JsonProperty("IdLocation")]
        public int IdLocation { get; }

        public Rating(int grade, string feedback, int idUserAccount, int idLocation)
        {
            this.Grade = grade;
            this.Feedback = feedback;
            this.IdUserAccount = idUserAccount;
            this.IdLocation = idLocation;
        }

        public override string ToString()
        {
            return "Note: " + Grade + " | " + Feedback;
        }
    }
}

File: models/rating.js
var method = Rating.prototype;

function Rating(Grade, Feedback,Date, IdUserAccount, IdLocation) {
    this.Grade = Grade;
    this.Feedback = Feedback;
    this.IdUserAccount = IdUserAccount;
    this.IdLocation = IdLocation;
	this.Date = Date;
}

module.exports = Rating;